src/main/java/seedu/address/model/person/DateRepaid.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.model.person;
+
+/**
+ * Represents the date of when the Person was sent to the whitelist in the address book, i.e. the date
+ * the Person repaid his debt completely.
+ * Guarantees: immutable;
+ */
+public class DateRepaid {
+
+    public static final String DATE_REPAID_PLACEHOLDER = "NOT REPAID";
```

Comment:
Refer to my pull request that added in Date Borrow field. That should give you an idea of how to use the Date class.